### PR TITLE
Brandon/election UI enhancements

### PR DESCRIPTION
**Pull request by brandonfancher**
### What's in this PR?
* Updates styles to match latest designs.
* Adds leaderboard list ordering animation example. 

### The Animation
The idea is that the frontend will poll (easy with `react-query`) periodically to get the latest vote totals for each participant in the round. When those totals update in the UI (or when you submit a new vote), the order of participants on the leaderboard will change along with the vote totals. You'll also see who is voting for whom.

This POC demonstrates that statically when you (the logged-in user) vote, as long as you're logged in as one of the people in the list.

https://user-images.githubusercontent.com/7911424/127066493-c0a5a763-c950-4b97-85f9-fb856d77051a.mp4

### CODE REVIEW

File: packages/webapp/src/_app/ui/expander.tsx [hunk 1]
```diff
@@ -5,27 +5,30 @@ import { Container } from "_app";
 
 interface ExpanderProps {
     header: React.ReactNode;
-    hideContentDivider?: boolean;
+    showContentDivider?: boolean;
```
Comment (cool-ant) [inline packages/webapp/src/_app/ui/expander.tsx:8]: 👍

File: packages/webapp/src/_app/ui/expander.tsx [hunk 2]
```diff
@@ -5,27 +5,30 @@ import { Container } from "_app";
...
     startExpanded?: boolean;
+    inactive?: boolean;
     children: React.ReactNode;
 }
 
 export const Expander = ({
     header,
-    hideContentDivider,
+    showContentDivider,
```
Comment (cool-ant) [inline packages/webapp/src/_app/ui/expander.tsx:16]: Does we use a particular form of this enough to provide a default value?

File: packages/webapp/src/elections/components/election-member-chips.tsx [hunk 3]
```diff
@@ -9,12 +9,17 @@ interface VotingMemberChipProps {
...
+    votingFor?: string;
 }
 
 export const VotingMemberChip = ({
     member,
     onSelect,
     isSelected,
+    votesReceived,
+    votingFor,
+    ...rest
```
Comment (cool-ant) [inline packages/webapp/src/elections/components/election-member-chips.tsx:22]: same here

File: packages/webapp/src/pages/election/ui.tsx [hunk 4]
```diff
@@ -223,16 +223,32 @@ const OngoingRoundSegment = ({
...
-                        onSelect={() => onSelectMember(member)}
-                    />
-                )}
-            </MembersGrid>
+            <Flipper flipKey={sortMembersByVotes}>
+                <MembersGrid members={sortMembersByVotes}>
+                    {(member) => (
+                        <Flipped
+                            key={`leaderboard-${member.account}`}
+                            flipId={`leaderboard-${member.account}`}
```
Comment (cool-ant) [inline packages/webapp/src/pages/election/ui.tsx:231]: really clean. Love how this reads

File: packages/webapp/src/_app/ui/expander.tsx (hunk 2 above)
Comment (brandonfancher) [reply packages/webapp/src/_app/ui/expander.tsx:16]: It defaults to `undefined` right now, which is falsy. BUT...I think adding a default value in does make it more explicit...easier to see what's actually going on. Added.

File: packages/webapp/src/_app/ui/generic-member-chip.tsx [hunk 5]
```diff
@@ -19,27 +19,30 @@ export const GenericMemberChip = ({
     actionComponent,
     contentComponent,
     footerComponent,
+    ...rest
```
[the remark this answers is not in the document]
Comment (brandonfancher) [reply packages/webapp/src/_app/ui/generic-member-chip.tsx:22]: Good suggestion. Updating to `...containerProps`.

**Pull request merged**